source/DemoView.cpp

```diff
@@ -65,9 +60,6 @@ DemoView::Draw(BRect updateRect)
 		fRadius += Settings.lwidth / 2 + Settings.lspace;
 		fColor.alpha += fAlphaMod;
 	}
-
-	ConstrainClippingRegion(NULL);
```

Comment:
After reading the BeBook link you gave me, I learned that drawing attributes are reset to their default values at the beginning of the Draw() function. I also tested it to make sure. So there is no need to reset background color and clipping region explicitly.